panel/widgets/slider.py

```diff
@@ -72,10 +75,10 @@ def _update_model(self, events, msg, root, model, doc, comm):
         return super()._update_model(events, msg, root, model, doc, comm)
 
 
-class ContinuousSlider(_SliderBase):
+class _ContinuousSlider(_SliderBase):
```

Comment:
That's an API breaking change which I believe isn't required for this documentation work.

Reply:
I will carve it out and make a separate PR. I did it because my understanding is that the ContinousSlider is used as a Parent class for other sliders and not exposed to users. There is no Gallery Reference Guide either.

I just did it because looking at the sliders module I looked a bit unfinished.

Reply:
Yes you're right I don't think this class should directly be used but since it's part of the public API (in Python the line between public and private API is somewhat blurry) as its name doesn't start with an underscore, we're less in our right to rename it as we'd like (some users may create their own widgets based on that class, who knows!). So we'd have to deprecate it first and then remove it in a future version.